source/scripts/index.js

```diff
@@ -0,0 +1,42 @@
+require('jquery');
+require('jquery-ui-bundle');
+
+class RangeSlider {
+    constructor(selector) {
+        this.selector = selector;
+        this.sliderSelector = $(selector);
+        this.min = this.sliderSelector.data("min");
+        this.max = this.sliderSelector.data("max");
+        this.mindefault = this.sliderSelector.data("mindefault");
+        this.maxdefault = this.sliderSelector.data("maxdefault");
+        this.step = this.sliderSelector.data("step");
+
+        this.init();
+    }
+
+    init() {
+        console.log(this.selector)
+        let s = this.selector;
+        this.sliderSelector.slider({
+            range: true,
+            min: this.min,
+            max: this.max,
+            values: [this.mindefault, this.maxdefault],
+            step: this.step,
+            slide: function (event, ui) {
+                $(s).find('[data-value-min]').text(ui.values[0]);
```

Comment:
`this.$slider.find(...)`

Reply:
i lepiej będzie zamiast za każdym razem _szukać_ tego elementu to w konstruktorze dodać `this.$minLabel = this.$slider.find(...);`, a w tym miejscu tylko `this.$labels.min.text(ui.values[0]);`